.travis.yml

```diff
@@ -13,15 +13,6 @@ jobs:
       env: TOXENV=py37-conan-latest
       dist: xenial
 
-    - stage: Conan Development - Linux
```

Comment:
Don't change this.

Reply:
Now I see that @uilianries provided this, Why? What this Travis has to do with the PR? I don't understand.

Reply:
Thinking again, I should keep it job. It's broken, but it's not problem, it's an warning say, "hey you need to update CPT, otherwise it won't work on next Conan release".

Reply:
Makes sense, I reverted CI change.